packages/web-components/src/image/image.options.ts

```diff
@@ -0,0 +1,46 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Border radius
+ * @public
+ */
+export const BorderRadius = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+  xlarge: 'x-large',
+} as const;
+/**
+ * Types for border radius
+ * @public
+ */
+export type BorderRadius = ValuesOf<typeof BorderRadius>;
+
+/**
+ * Image fit
+ * @public
+ */
+export const ImageFit = {
```

Comment:
Is it possible that other components have `ImageFit` (@miroslavstastny do you think this needs namespacing or are we safe here?) It is the fit of the image....

Reply:
to be clear here, I think this is find, but I want to make sure we don't need to be doing `ImageImageFit` :/

Reply:
Closing this as I've answered my own - attribute is `fit`, so this is already namespaced. Move along.

Reply:
These are scoped to this component. How will these named attributes/types impact other components whose attributes are named the same?

Reply:
No change to this one, only to `BorderRadius` above